packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -184,31 +183,33 @@ const Trade = observer(() => {
                             <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
                             <TradeChart topWidgets={topWidgets} is_accumulator={is_accumulator} />
                         </div>
-                    </DesktopWrapper>
-                    <MobileWrapper>
-                        <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
-                        <SwipeableWrapper
-                            className={classNames({ 'vanilla-trade-chart': is_vanilla })}
-                            is_disabled={
-                                !show_digits_stats ||
-                                !is_trade_enabled ||
-                                !form_components.length ||
-                                is_chart_loading ||
-                                should_show_active_symbols_loading
-                            }
-                            is_swipe_disabled={swipe_index === 1}
-                            onChange={onChangeSwipeableIndex}
-                            should_elevate_navigation={should_elevate_navigation}
-                        >
-                            {show_digits_stats && <DigitsWidget digits={digits} tick={tick} />}
-                            <TradeChart
-                                bottomWidgets={show_digits_stats ? bottomWidgets : undefined}
-                                has_barrier={has_barrier}
-                                is_accumulator={is_accumulator}
-                                topWidgets={topWidgets}
-                            />
-                        </SwipeableWrapper>
-                    </MobileWrapper>
+                    )}
+                    {isMobile && (
```

Comment:
perhaps we could use ternary here? will it work? :)
isMobile ? <div... : <React.Fragment...